scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts

```diff
@@ -21,6 +28,55 @@ const BATCH_SIZE = 100
 const axiosInstance = axios.create()
 axiosRetry(axiosInstance, { retries: 5, retryDelay: axiosRetry.exponentialDelay })
 
+const manualRelatedAssetIndex: Record<AssetId, AssetId[]> = {
+  [ethAssetId]: [
+    optimismAssetId,
+    arbitrumAssetId,
+    arbitrumNovaAssetId,
+    // WETH on Ethereum
+    'eip155:1/erc20:0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+    // WETH on Gnosis
+    'eip155:100/erc20:0x6a023ccd1ff6f2045c3309768ead9e68f978f6e1',
+    // WETH on Polygon
+    'eip155:137/erc20:0x7ceb23fd6bc0add59e62ac25578270cff1b9f619',
+    // WETH on Arbitrum One
+    'eip155:42161/erc20:0x82af49447d8a07e3bd95bd0d56f35241523fbab1',
+    // WETH on Arbitrum Nova
+    'eip155:42170/erc20:0x722e8bdd2ce80a4422e880164f2079488e115365',
+    // WETH on Avalanche
+    'eip155:43114/erc20:0x49d5c2bdffac6ce2bfdb6640f4f80f226bc10bab',
+    // WETH on BSC
+    'eip155:56/bep20:0x2170ed0880ac9a755fd29b2688956bd959f933f8',
+    // WETH on Optimism
+    'eip155:10/erc20:0x4200000000000000000000000000000000000006',
+  ],
+}
+
+export const getManualRelatedAssetIds = (
+  assetId: AssetId,
+): { relatedAssetIds: AssetId[]; relatedAssetKey: AssetId } | undefined => {
+  // assetId is the primary implementation for the related assets, which makes it pretty easy, just access the property and voila
+  if (manualRelatedAssetIndex[assetId]) {
+    return {
+      relatedAssetIds: manualRelatedAssetIndex[assetId],
+      relatedAssetKey: assetId,
+    }
+  }
+
+  // assetId isn't the primary implementation, but may be one of the related assets
+  for (const [primaryAssetId, relatedAssetIds] of Object.entries(manualRelatedAssetIndex)) {
+    if (relatedAssetIds.includes(assetId)) {
+      return {
+        relatedAssetIds: [...relatedAssetIds, primaryAssetId],
```

Comment:
we don't need to add the primary impl asset to the related assets
```suggestion
        relatedAssetIds,
```

Reply:
might as well update `primaryAssetId` -> `relatedAssetKey` and go full shortened syntax mode :muscle: :joy:

Reply:
Very valid, `relatedAssetKey` should take care of the primary implementation without any need for us to push it to the `relatedAssetIds` 🙏🏽 a5fbbff84d71f27ae359b8f91339c369527091e